src/main/java/uk/gov/hmcts/darts/datamanagement/service/impl/InboundAnnotationTranscriptionDeleterProcessorImpl.java

```diff
@@ -61,7 +61,7 @@ public List<Integer> markForDeletion(int hourBeforeCurrentDate) {
             currentTimeHelper.currentOffsetDateTime()
         );
 
-        log.debug("Records have been marked for deletion");
+        recordsMarkedForDeletion.stream().forEach(eodId -> log.info("Set status of unstructured EOD {} to be marked for deletion", eodId));
```

Comment:
Will be worth wrapping this in a if(log.isInfoEnabled() {...}

Reply:
As far as I know log level will never be less than info so its not really required unless we want to reduce the strings on the stack on start up. I am outputting individual eod id's as its easier for support to read them rather than a long string seperated by commas.

Reply:
I'm happy with the above :)